drivers/modem/Kconfig

```diff
@@ -14,6 +14,7 @@ if MODEM
 config MODEM_U_BLOX
 	bool "Enable u-blox modem driver"
 	default n
+	depends on ALLOW_BSD_COMPONENTS
```

Comment:
It will break your "own" Sony Spresense "lte" config.

Reply:
@acassis we have to make a basic build that is APACHE only then each developer can consider if they want to use BSD licenses